dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/OAuth20EndpointServices.java

```diff
@@ -342,7 +342,15 @@ protected void handleEndpointRequest(HttpServletRequest request,
                         "com.ibm.ws.security.oauth20.web.OAuth20EndpointServices", "324", this);
             }
             boolean suppressBasicAuthChallenge = isBrowserWithBasicAuth; // ui must NOT log in using basic auth, so logout function will work.
-            WebUtils.sendErrorJSON(response, e.getHttpStatus(), e.getErrorCode(), e.getErrorDescription(), suppressBasicAuthChallenge);
+            /*
+             * BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_MALFORMED_REQUEST");
+             * Tr.error(tc, errorMsg.getServerErrorMessage());
+             * throw new OidcServerException(errorMsg.getBrowserErrorMessage(), OIDCConstants.ERROR_INVALID_CLIENT_METADATA, HttpServletResponse.SC_BAD_REQUEST);
+             */
+
+            // BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(e.getBrowserServerLogMessage().getTraceComponent(), request.getLocales(), e.getBrowserServerLogMessage().getMessageKey());
+            e.getBrowserServerLogMessage().setTraceComponent(tc);
```

Comment:
We don't want this call. The `BrowserAndServerLogMessage` object within the exception was created with the `TraceComponent` object that points to the file with the right NLS message. If you do `e.getBrowserServerLogMessage().setTraceComponent(tc);` right here, you've just overridden that `TraceComponent` object with one that might point to a different NLS message file.